circle.yml

```diff
@@ -25,11 +25,11 @@ test:
     # 0) runs unit tests w/ H2 local DB. Runs against H2, Mongo, MySQL
     # 1) runs unit tests w/ Postgres local DB. Runs against H2, SQL Server
     # 2) runs unit tests w/ MySQL local DB. Runs against H2, Postgres, SQLite
-    # 3) runs Eastwood linter
-    # 4) Bikeshed linter
+    # 3) runs unit tests w/ H2 local DB. Runs against H2, Redshift
```

Comment:
I'd like to get some ideas going on how we can improve this pattern.  I don't think it makes sense to run an instance of every database we can think of 24/7 just so that we can run unit tests.  Feels like we are just now breaking the threshold where it makes sense to evaluate some other options.